src/loadgenerator/requirements.txt

```diff
@@ -19,17 +20,20 @@ locust==2.18.2
 locust_plugins==3.4.0
 markupsafe==2.1.3
 msgpack==1.0.7
-opentelemetry-api==1.23.0
-opentelemetry-exporter-otlp-proto-grpc==1.23.0
-opentelemetry-instrumentation==0.44b0
-opentelemetry-instrumentation-jinja2==0.44b0
-opentelemetry-instrumentation-requests==0.44b0
-opentelemetry-instrumentation-system-metrics==0.44b0
-opentelemetry-instrumentation-urllib3==0.44b0
-opentelemetry-proto==1.23.0
-opentelemetry-sdk==1.23.0
-opentelemetry-semantic-conventions==0.44b0
-opentelemetry-util-http==0.44b0
+opentelemetry-api==1.25.0
+opentelemetry-exporter-otlp==1.25.0
+opentelemetry-exporter-otlp-proto-common==1.25.0
+opentelemetry-exporter-otlp-proto-grpc==1.25.0
+opentelemetry-exporter-otlp-proto-http==1.25.0
```

Comment:
Will these dependencies create extra signals? I am wondering if it would make sense adding them into the upstream repository instead

Reply:
Nope, there are just one more dependency of our distro that they don't install